### PR TITLE
Update next-steps pages

### PR DESCRIPTION
**Pull request by cnunciato**
Updates the next-steps pages at the end of each Getting Started tutorial to include links to Learn, Docs, How-to guides, and the Blog.

### CODE REVIEW

**Comment by susanev** [comment]
can yall speak to why its important to present the links in paragraph form in place of a list?

File: themes/default/content/docs/get-started/kubernetes/next-steps.md [hunk 1]
```diff
@@ -13,28 +13,28 @@ menu:
...
+{{< get-started-next-step path="/learn/pulumi-fundamentals" label="Learn Pulumi Fundamentals" ref="gs-k8s-learn" >}}
 
-- [AWS EKS - Hello World!]({{< relref "/registry/packages/kubernetes/how-to-guides/eks" >}})
-- [Azure AKS - Hello World!]({{< relref "/registry/packages/kubernetes/how-to-guides/aks" >}})
-- [Google Cloud GKE - Hello World!]({{< relref "/registry/packages/kubernetes/how-to-guides/gke" >}})
+## How-to Guides
 
-## Workloads
+Explore our how-to guides if you're looking for examples of specific architectures or application stacks. These guides are available in all Pulumi languages and cover creating managed Kubernetes clusters across major cloud providere ([AWS]({{< relref "/registry/packages/kubernetes/how-to-guides/eks" >}}), [Azure]({{< relref "/registry/packages/kubernetes/how-to-guides/aks" >}}), [Google Cloud]({{< relref "/registry/packages/kubernetes/how-to-guides/gke" >}})) and deploying app workloads to running Kubernetes clusters ([WordPress Helm Chart]({{< relref "/registry/packages/kubernetes/how-to-guides/wordpress-chart" >}}), [Stateless App Deployment]({{< relref "/registry/packages/kubernetes/how-to-guides/stateless-app" >}})).
```
[the remark this answers is not in the document]
Comment (cnunciato) [reply themes/default/content/docs/get-started/kubernetes/next-steps.md:26]: Ah, yeah -- that sounds better. I'll push a change.

**Comment by susanev** [comment]
@cnunciato all of them. am i reading this wrong? looks like we are moving from a list of links to a paragraph of links.

**Comment by aaronkao** [comment]
The current page was just a bunch of links that dump you into examples. It didn't really give much guidance on "what should I do next". There are many different next steps for people depending on their learning style or sophistication with the cloud and/or IaC. By presenting the 4 different options with some explanation copy, it should give the reader a good sense of the options they can take and invite them to take at least one of the offered steps.

**Comment by susanev** [comment]
Okay. Do we have tracking setup? Would be good to watch what actions people take from these pages in comparison to actions people took before these pages and iterate more if there is any concerning results from the paragraph presentation.

**Comment by cnunciato** [comment]
We do have tracking set up on the buttons, yep.

**Comment by susanev** [comment]
ah, i was hoping we had something to compare it to.

**Comment by joeduffy** [comment]
Agree with @susanev, thoughts on tracking whether we improved conversion?

**Comment by cnunciato** [comment]
We definitely want to track how well these changes perform, yes. Before this change, though, we didn't have any additional click tracking enabled on these pages, so we won't have direct comparisons there, unfortunately. (I added click tracking in in this pass, however, so going forward, we will.) 

I'm sure we'll be able to compare whatever numbers we've been gathering up to now for engagement with these pages with these new changes, though. I'll check with @zchase to see what our options are in that regard.

**Pull request merged**